mne/io/base.py

```diff
@@ -1745,9 +1745,15 @@ def _repr_html_(self, caption=None):
         basenames = [
             os.path.basename(f) for f in self._filenames if f is not None
         ]
-        m, s = divmod(self._last_time - self.first_time, 60)
-        h, m = divmod(m, 60)
-        duration = f'{int(h):02d}:{int(m):02d}:{int(s):02d}'
+
+        # https://stackoverflow.com/a/10981895
+        duration = timedelta(seconds=self._last_time - self.first_time)
```

Comment:
We use a different calculation than in `__repr__()` here. In `__repr__()`, we use `self.times[-1]`, which I believe is a bug.

cc @larsoner @agramfort

Reply:
Never mind, it appears `raw.times` is already adjusted for `first_time`.